docker-compose.yaml

```diff
@@ -1,17 +1,28 @@
 version: '3.4'
 
 services:
+    db:
+        image: postgres:11.2-alpine
     app:
+        &app
         build:
             context: .
             target: prod
         environment:
             APP_SECRET:
             DEFAULT_LOCALE:
             SERVICE_NAME:
+            DATABASE_URL: pgsql://postgres:@db:5432/postgres
         image: libero/content-store:${IMAGE_TAG:-master}
-        volumes:
-            - ./.docker/data:/app/data/
+        depends_on:
+            - db
+    db-schema:
+        <<: *app
+        command: >
```

Comment:
another option for `doctrine:migrations:migrate` is to execute it as part of `app`'s `command` so that it doesn't start until there is a database ready for it

Reply:
:+1: a74250b